### ISSUE TITLE
Elements get type "any" instead of the correct one

### ISSUE THREAD

**Issue by nachtigall-83** [horiuchi/dtsgenerator#448]
Hi,

When converting a json schema that has several interfaces and one interface is used inside another one, then in the latter the elements get type "any" instead of the real type, like "number" or "string".
Example: 
Interface Account is used inside GetBillAccountResponse:
export interface Account {
        /**
         * accountId
         */
        accountId?: **number**;
        /**
         * bskId
         */
        bskId: **number**; 
       ...

export interface GetBillAccountResponse {
        /**
         * account
         */
        account?: {
            /**
             * accountId
             */
            accountId?: **any**;
            /**
             * bskId
             */
            bskId: **any**;
            ....

I expect in GetBillAccoutResponse the elements also have the correct types and not "any".

And could you please add an option to not have all the comments? I don't need them.

Thanks a lot!

**Comment by horiuchi** [comment]
Please show me the actual JSON Schema that the problem reproduces.

**Comment by nachtigall-83** [comment]
I attached a zip with the json schema and the result I get with 
dtsgen --out src/types/AccountRead_min.dtsgen.d.ts src/resources/AccountRead_min.schema.json

[types.zip](https://github.com/horiuchi/dtsgenerator/files/5312082/types.zip)

I expect that the element "account" (line 99) in the interface "GetBillAccountResponse" (line 95) looks like the interface "Account" starting from line 5 or is somehow a reference to it. 

Thanks!

**Comment by horiuchi** [comment]
I repro this issue by this schema. I will investigate it.
```
id: '#'
definitions:
  A:
    properties:
      s:
        allOf:
          - type: string
  B:
    properties:
      ref:
        allOf:
          - $ref: '#/definitions/A'
properties:
  bref:
    $ref: '#/definitions/B'
```

**Comment by horiuchi** [comment]
Released by v3.3.1
